tensorflow/python/feature_column/feature_column.py

```diff
@@ -2372,9 +2370,11 @@ def _get_dense_tensor(self, inputs, weight_collections=None, trainable=None):
     return inputs.get(self)
 
 
-class _BucketizedColumn(_DenseColumn, _CategoricalColumn,
-                        collections.namedtuple('_BucketizedColumn', [
-                            'source_column', 'boundaries'])):
+class _BucketizedColumn(
+    _DenseColumn,
+    _CategoricalColumn,
+    _SequenceDenseColumn,
```

Comment:
Note that it is currently impossible to have a sequence column which behaves as both dense and categorical. See comment in #18975.